manticore/core/workspace.py

```diff
@@ -401,7 +410,8 @@ def __init__(self, desc=None):
         :param desc: A descriptor ('type:uri') of where to write output.
         """
         self._named_key_prefix = 'test'
-        self._store = _create_store(desc)
+        self._descriptor = desc
```

Comment:
We can't because `desc` is `None` when we should create a new tmp workspace. I updated it to cache the descriptor so we don't recalc every time.